site/src/pages/docs/client-factory-builder.mdx

```diff
@@ -0,0 +1,164 @@
+# Building a client using ClientFactoryBuilder
```

Comment:
How about merging this document into the subsections in `client-basics`?

Reply:
I think merging this document into the basic part would make too long document to read. Furthermore, I suppose basic document needs to be proper length for beginner.

Also, current length is a bit long and the content itself is not proper for basics.